jobs/check-aws-regions.Jenkinsfile

```diff
@@ -26,7 +26,7 @@ cosaPod(serviceAccount: "jenkins"){
     }
     if (disabled_regions != "None") {
         warn("Disabled AWS regions detected: ${disabled_regions}")
-        pipeutils.trySlackSend(message: ":aws: aws-regions-disabled :jenkins:<${env.BUILD_URL}|${env.BUILD_NUMBER}> ${disabled_regions}")
+        pipeutils.trySlackSend(message: ":aws: check-aws-regions #${env.BUILD_NUMBER} <${env.BUILD_URL}|:jenkins:> <${env.RUN_DISPLAY_URL}|:ocean:> detected disabled regions: ${disabled_regions}")
```

Comment:
Was this tested? When we added the blue ocean link to the other notifications, I couldn't find any documentation that supported  this. I hope it works!

Reply:
I suggested it in https://github.com/coreos/fedora-coreos-pipeline/pull/899#discussion_r1276742315 but I have not tested it.

Which part are you worried about working? the hyperlinking of an emoji?

Reply:
Yes, I didn't think slack allowed this. It's worth a try!

Reply:
I am not able to test it on the stage pipeline. Probably we can merge this and see how it works and then make the changes to other jobs?

Reply:
Yeah that works for me. I wasn't too happy with repeating the build number in the other jobs so this would be ideal!